### ISSUE TITLE
CSV/TSV Output

### ISSUE THREAD

**Issue by mhbeals** [interedition/collatex#38]
The ability to output in CSV and/or TSV formats

**Comment by djbpitt** [comment]
@mhbeals We're reviewing open issues and wondering whether you were still interested in this feature. If so, have I understood correctly that what you'd like is output similar to the alignment table, but with commas or tabs instead of the ASCII table borders? Thanks!

**Comment by ebeshero** [comment]
@djbpitt Observing from afar, the projects I'm involved in will benefit from this feature as well. I've recently been adapting collateX XML output with XSLT to produce output text with TSV to import into various spreadsheet formats.

**Comment by mhbeals** [comment]
Yes, that was the idea. It would be a really helpful feature!

**Comment by djbpitt** [comment]
@mhbeals I've added an output type of "csv" at https://github.com/djbpitt/collatex/tree/csv ("csv" branch of my fork). The Python csv writer has some configuration options, listed at https://docs.python.org/3/library/csv.html#dialects-and-formatting-parameters, and I've left them all unspecified, that is, accepted the defaults. I'll add support for tsv once we've confirmed that csv is working correctly, but in the meanwhile, I would be grateful if you could please let me know whether it does what you want, and whether you think we should set any of the formatting parameters to any value other than the default. 

To test, clone my fork of the collatex repo, `cd` to `collatex/collatex-pythonport`, temporarily uninstall your system version of collatex with `pip uninstall collatex`, and then run a test script, while you're located inside `collatex/collatex-pythonport`, either from a file or within a Jupyter Notebook. When I run:
```python
%reload_ext autoreload
%autoreload 2
from collatex import *
collation = Collation()
collation.add_plain_witness("A", "The big gray koala")
collation.add_plain_witness("B", "The grey koala")
print(collate(collation, output="csv"))
```
the output is:
```
A,The ,big gray ,koala
B,The ,grey ,koala
```

**Comment by djbpitt** [comment]
@mhbeals "csv" and "tsv" output methods have been included in CollateX Python 2.2.
